diff/cimfs/cimfs.go

```diff
@@ -0,0 +1,198 @@
+// +build windows
```

Comment:
Could we have a common package that this and the windows differ could share? The code is almost entirely the same besides the option passed to apply, e.g. diff/windows/cimfs and diff/windows/windows and have common code in diff/windows

Reply:
Kind of like what you did for our snapshotters

Reply:
I like the idea but I would prefer to do this in the next PR (https://github.com/ambarve/containerd/pull/1) just to avoid getting into weird rebase merge conflicts. Is that okay?

Reply:
Sure that's fine